internal/common/voting/election.go

```diff
@@ -24,60 +24,15 @@ func (e *Election) ProposeElection(role shared.Role, method shared.ElectionVotin
 // OpenBallot sets the islands eligible to vote.
 func (e *Election) OpenBallot(clientIDs []shared.ClientID, clientMap map[shared.ClientID]baseclient.Client) {
 	e.voterList = clientIDs
-	//Initialize candidate list.
-	e.candidateList = e.voterList
-	//Get current President, Judge and Speaker IDs. 
-	var currentPresidentID shared.ClientID
-	var currentJudgeID shared.ClientID
-	var currentSpeakerID shared.ClientID
-	for i := 0; i < len(e.voterList); i++ {
-		if clientMap[e.voterList[i]].MonitorIIGORole(shared.President) == true {
-			currentPresidentID = e.voterList[i]
-		} else if clientMap[e.voterList[i]].MonitorIIGORole(shared.Judge) == true {
-			currentJudgeID = e.voterList[i]
-		} else if clientMap[e.voterList[i]].MonitorIIGORole(shared.Speaker) == true {
-			currentSpeakerID = e.voterList[i]
-		}
-	}
-	currentRoleIDList := []shared.ClientID{currentPresidentID, currentJudgeID, currentSpeakerID}
-
-	//Determine role to be changed.
-	var roleToBeChanged shared.ClientID
-	switch e.roleToElect {
-	case shared.President:
-		roleToBeChanged = currentRoleIDList[0]
-	case shared.Judge:
-		roleToBeChanged = currentRoleIDList[1]
-	case shared.Speaker:
-		roleToBeChanged = currentRoleIDList[2]
-	}
-
-	//Delete the voters from initial candidate list who already have a President, Speaker or Judge. 
-	var roleIDIndex []int
-	var roleToBeChangedIndex int
-	for i:=0;i<len(currentRoleIDList);i++ {
-		if currentRoleIDList[i] == roleToBeChanged {
-			roleToBeChangedIndex = i
-		}
-	}
-	for i := 0; i < len(currentRoleIDList); i++ {
-		if i != roleToBeChangedIndex {
-			for j := 0; j < len(e.candidateList); j++ {
-				if e.candidateList[j] == currentRoleIDList[i] {
-					roleIDIndex = append(roleIDIndex, j)
-				}
+	//Get candidate list.
+	allClients := []shared.ClientID{shared.Team1, shared.Team2, shared.Team3, shared.Team4, shared.Team5, shared.Team6}
+	for i:=0;i<len(allClients);i++ {
+		for island, _ := range clientMap {
+			if island == allClients[i] {
+				e.candidateList = append(e.candidateList, island)
 			}
 		}
 	}
```

Comment:
```suggestion
		for island, _ := range clientMap {
				e.candidateList = append(e.candidateList, island)
		}
```

I think this works too...?